packages/core/src/drivers/db/models/file.js

```diff
@@ -0,0 +1,68 @@
+const { Model, DataTypes, Sequelize } = require('sequelize');
+
+const FILE_TABLE = 'files';
+
+const FileSchema = {
+  id: {
+    allowNull: false,
+    autoIncrement: true,
+    primaryKey: true,
+    type: DataTypes.INTEGER,
+  },
+  tipoEntidad: {
+    allowNull: false,
+    type: DataTypes.INTEGER,
+    field: 'tipo_entidad',
+  },
+  entidadId: {
+    allowNull: false,
+    type: DataTypes.INTEGER,
+    field: 'entidad_id',
+  },
```

Comment:
Why do you have tipoEntidad and entidadId? tipoEntidad should be in entidades table